miden-lib/asm/miden/kernel_proc_offsets.masm

```diff
@@ -1,96 +1,132 @@
 # OFFSET CONSTANTS
 # -------------------------------------------------------------------------------------------------
 
-# Current account
-const.ACCOUNT_VAULT_ADD_ASSET_OFFSET=0
-const.ACCOUNT_VAULT_GET_BALANCE_OFFSET=1
-const.ACCOUNT_VAULT_HAS_NON_FUNGIBLE_ASSET_OFFSET=2
-const.ACCOUNT_VAULT_REMOVE_ASSET_OFFSET=3
-const.GET_ACCOUNT_ID_OFFSET=4
-const.GET_ACCOUNT_ITEM_OFFSET=5
-const.GET_ACCOUNT_MAP_ITEM_OFFSET=6
-const.GET_ACCOUNT_NONCE_OFFSET=7
-const.GET_ACCOUNT_VAULT_COMMITMENT_OFFSET=8
-const.GET_CURRENT_ACCOUNT_HASH_OFFSET=9
-const.GET_INITIAL_ACCOUNT_HASH_OFFSET=10
-const.INCR_ACCOUNT_NONCE_OFFSET=11
-const.SET_ACCOUNT_CODE_OFFSET=12
-const.SET_ACCOUNT_ITEM_OFFSET=13
-const.SET_ACCOUNT_MAP_ITEM_OFFSET=14
-
-# Current faucet
-const.BURN_ASSET_OFFSET=15
-const.GET_FUNGIBLE_FAUCET_TOTAL_ISSUANCE_OFFSET=16
-const.MINT_ASSET_OFFSET=17
-
-# Current note
-const.ADD_ASSET_TO_NOTE_OFFSET=18
-const.CREATE_NOTE_OFFSET=19
-const.GET_INPUT_NOTES_COMMITMENT_OFFSET=20
-const.GET_NOTE_ASSETS_INFO_OFFSET=21
-const.GET_NOTE_INPUTS_HASH_OFFSET=22
-const.GET_NOTE_SENDER_OFFSET=23
-const.GET_NOTE_SERIAL_NUMBER_OFFSET=24
-const.GET_SCRIPT_HASH_OFFSET=25
-const.GET_OUTPUT_NOTES_COMMITMENT_OFFSET=26
-
-# Transaction
-const.GET_BLOCK_HASH_OFFSET=27
-const.GET_BLOCK_NUMBER_OFFSET=28
-const.START_FOREIGN_CONTEXT_OFFSET=29
-const.END_FOREIGN_CONTEXT_OFFSET=30
-const.UPDATE_EXPIRATION_BLOCK_NUM_OFFSET=31
-const.GET_EXPIRATION_DELTA_OFFSET=32
+### Account #####################################
+# Entire account hash
+const.ACCOUNT_GET_CURRENT_HASH_OFFSET=0
+const.ACCOUNT_GET_INITIAL_HASH_OFFSET=1
+
+# ID
+const.ACCOUNT_GET_ID_OFFSET=2
+
+# Nonce
+const.ACCOUNT_GET_NONCE_OFFSET=3                 # accessor
+const.ACCOUNT_INCR_NONCE_OFFSET=4                # mutator
+
+# Procedures
+## get account code is missing
+const.ACCOUNT_PROCEDURES_SET_CODE_OFFSET=5
```

Comment:
Should we call this `ACCOUNT_SET_CODE_COMMITMENT_OFFSET`? More generally though, I wonder if we should just remove this procedure and all associated code as I don't think we currently support updating account code and the way it'll eventually work will probably be different from now (e.g., we'll also need to update storage slot data).

But, I'd add `ACCOUNT_GET_CODE_COMMITMENT_OFFSET` and create a procedure for it. The procedure would be pretty simple (i.e., just return the code commitment of the current account).